src/extract/strategy_smart_by_first_node_and_tags.cpp

```diff
@@ -215,7 +230,7 @@ namespace strategy_smart_by_first_node_and_tags {
                         if (e->way_ids.get(member.positive_ref())) {
                             if (wanted_members == 0) {
                                 e->relation_ids.set(relation.positive_id());
-                                if (strategy().check_type(relation) && strategy().check_tags(relation)) {
+                                if (strategy().check_type(relation) || strategy().check_tags(relation)) {
```

Comment:
I am not sure why we are seeing this change here.
It was already done in 4213ca521c494aafb25dd3410c2902e6ec41f330.

Reply:
On your target branch this commit was changed to https://github.com/andrzejkurczewski-tomtom/osmium-tool/pull/2/commits/0c29df6e567f0bb1e3ba352a1e38b2fda409b34d so https://github.com/andrzejkurczewski-tomtom/osmium-tool/commit/4213ca521c494aafb25dd3410c2902e6ec41f330 is only on your branch

Reply:
Note that this other branch is a development branch, that can be reviewed here: https://github.com/andrzejkurczewski-tomtom/osmium-tool/pull/2
I didn't have time to look into it yet, feel free to do so, maybe we can merge both :)

Reply:
Thanks, I rebased my branch.